### DISCUSSION TITLE
Default options

### DISCUSSION THREAD

**Discussion by leo-arch** [leo-arch/clifm#19] [Q&A]
# What should be CliFM's default options?

For example, I always enable hidden files, but it seems that most file managers turn them off by default. The same goes for non-sensitive TAB completion: it's a quite helpful feature, but currently disabled by default. We have plenty of options and it would be nice to have good sane defaults.

Take a look at the current configuration file and write a list with what you think should be the default values, so that we can start the debate.

Thanks in advance!
Archcrack.

**Comment by tuerda** [comment]
Hey there. The options I might consider changing from their current defaults are:
- cdonquit
- casesensitivepathcomp
- terminalcmd (it has urxvt hardcoded now, rather than checking for the user default . . . either that or it found that mine was urxvt and wrote that into the config file, which seems odd.)

There were also several options which did not have an explanation and it was not immediately obvious what they did:
- splashscreen
- longviewmode
- logcmds
- tips
- casesensitivelist
- casesensitivedirjump
- pager

**Reply by leo-arch** [reply]
Thanks @tuerda, that was quick!

* The value for `TerminalCmd` is taken from `DEFAULT_TERM_CMD`, which is hardcoded to "xterm -e" in `helpers.h`:
```c
#define DEFAULT_TERM_CMD "xterm -e"
```
So, it shouldn't be set to `urxvt` if you didn't do it yourself. To make sure, create a new profile (`-P NAME` from the command line or `p add NAME` from within CliFM) and check the value of `TerminalCmd` in the corresponding profile directory.

* `CaseSensitivePathComp`: Absolutely. For me, it's a must.
* `CdOnQuit`: This one needs to be set up sourcing a little script before running CliFM. See [cd on quit](https://github.com/leo-arch/clifm/wiki/Advanced#cd-on-quit). This is why it is not enabled by default: it won't work out of the box.

As to the remaining options, I will just add a simple description.

**Reply by leo-arch** [reply]
Done: new defaults (case insensitive is the default now) and explanations for options in the config file.
